### PR TITLE
Issue#174 If then block should start with `{` and end with `}`

### PR DESCRIPTION
**Pull request by yokotaso**
Thank you for developing wonderful tool!

I found this issue, 
https://github.com/shyiko/ktlint/issues/174

And then, I tried to develop IfThenBlockWithCurlyBraceRule.
If you have a free time, Cloud you review it?

### CODE REVIEW

File: ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/IfThenBlockWithCurlyBraceRule.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.jetbrains.kotlin.lexer.KtTokens
+
+class IfThenBlockWithCurlyBraceRule : Rule("if-then-without-curly-brace-rule") {
+    override fun visit(node: ASTNode, autoCorrect: Boolean, emit: (offset: Int, errorMessage: String, canBeAutoCorrected: Boolean) -> Unit) {
+
+
+        if (node.elementType == KtNodeTypes.THEN) {
+            assert(node.treeParent.elementType == KtNodeTypes.IF)
+            // Check Whether Block Element start with `{`
+            if (node.firstChildNode.firstChildNode.elementType != KtTokens.LBRACE) {
```
Comment (yokotaso) [inline ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/IfThenBlockWithCurlyBraceRule.kt]: Are thre needed nullable check?

File: ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/IfThenBlockWithCurlyBraceRule.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                emit(node.firstChildNode.startOffset, "if-then block should start with `{`", true)
+                if (autoCorrect) {
+                    (node.firstChildNode.firstChildNode as TreeElement).rawInsertBeforeMe(LeafPsiElement(KtTokens.RBRACE, "{"))
+                }
+            }
+            // Check Whether Block Element end with `}`
+            if (node.lastChildNode.lastChildNode.elementType != KtTokens.RBRACE) {
```
Comment (yokotaso) [inline ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/IfThenBlockWithCurlyBraceRule.kt]: Is it nullable check needed?

**Comment by shyiko** [comment]
I'm sorry, Tomoya. I somehow managed to miss the notification about this PR. Merging in! :bowing_man:

**Comment by yokotaso** [comment]
@shyiko Thank you for merging PR! 🙇

**Pull request merged**